adafruit_hid/us_keyboard_layout.py

```diff
@@ -0,0 +1,218 @@
+# The MIT License (MIT)
```

Comment:
Do you want the class name to match for keyboard_layout_us.py? KeyboardLayoutUS?